def _check_error(self, resp):
    if (resp.error and (resp.error.error_code != 0)):
        raise VoldemortException(resp.error.error_message, resp.error.error_code)
